web-client/src/app/views/pay/pay.page.stories.ts

```diff
@@ -21,6 +22,7 @@ type Args = { receiverAddress: string } & SessionState;
 
 const Template: Story<Args> = ({ receiverAddress, ...state }: Args) => ({
   moduleMetadata: {
+    declarations: [PayFromToComponent],
```

Comment:
This declaration should be removed: it conflicts with the one in `SharedModule`.

(This doesn't actually present in Chromatic, but it makes the story fail when running Storybook locally.)

Reply:
e63e060 should resolve this.